### DISCUSSION TITLE
Make affix tier coloring respect the item's ilvl

### DISCUSSION THREAD

**Discussion by worldendDominator** [Lailloken/Exile-UI#150] [Ideas & Feedback]
My idea is to color-code affix tiers in item info popup according to what's possible to roll on the item's level, not at maximum item level. For example, if a body armor with ilvl 25 rolls T9 +life mod (which requires ilvl 24), it should be colored as highest possible (because you can't roll any higher tier). This would be helpful for picking items during leveling.

It can also be an option to switch between current and suggested behavior, in case it could mess with crafting.

**Comment by Lailloken** [comment]
Yeah, that was my initial plan for a feature geared towards league-starting, but the datamined databases are quite messy when it comes to mod-names and -tagging. I ran out of time before I could find a reliable way of compiling the required data and especially double-checking if everything is correct. The databases still include retired/legacy mods that often share the same name/ID with current ones, and then are simply tagged as "old" or "donotuse" somewhere in the properties.

I then chose to prioritize the item-comparison feature because it more immediately helps with leveling/league-start and relies only on the information that's available on the item itself. Knowing that an item has one or two maxed out mods doesn't help much if it's much worse in other aspects than what you currently have equipped.

But what you suggest is definitely still on the roadmap.
